install/README.md

```diff
@@ -41,3 +41,62 @@ wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O
 ```
 wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash -s <Version>
 ```
+
+## For Users with Poor Network Conditions
+
+You can download resources from a mirror instead of from Github.
+
+### Windows
+
+- Create a `CustomAssetFactory` function to define what the release asset url you want to use
+- Pass `DaprReleaseJsonUrl` as the json result of all releases as <https://api.github.com/repos/dapr/cli/releases>
+- You could use cdn.jsdelivr.net global CDN for your location to download install.ps1
+
+### Get the latest stable version
+
+For example, if you are in Chinese mainland, you could use:
+
+- Gitee.com to get latest release.json
+- cnpmjs.org hosted by Alibaba for assets
+- cdn.jsdelivr.net global CDN for install.ps1
+
+```powershell
+function CustomAssetFactory {
+    param (
+        $release
+    )
+    [hashtable]$return = @{}
+    $return.url = "https://github.com.cnpmjs.org/dapr/cli/releases/download/$($release.tag_name)/dapr_windows_amd64.zip"
+    $return.name = "dapr_windows_amd64.zip"
+    $return
+}
+$params = @{
+    CustomAssetFactory = ${function:CustomAssetFactory};
+    DaprReleaseJsonUrl    = "https://gitee.com/dapr-cn/dapr-bin-mirror/raw/main/cli/releases.json";
+}
+$script=iwr -useb https://cdn.jsdelivr.net/gh/dapr/cli/installer/install.ps1;
+$block=[ScriptBlock]::Create(".{$script} $(&{$args} @params)");
+Invoke-Command -ScriptBlock $block
+```
+
+### Get the specific version
+
+```powershell
+function CustomAssetFactory {
+    param (
+        $release
+    )
+    [hashtable]$return = @{}
+    $return.url = "https://github.com.cnpmjs.org/dapr/cli/releases/download/$($release.tag_name)/dapr_windows_amd64.zip"
+    $return.name = "dapr_windows_amd64.zip"
+    $return
+}
+$params = @{
+    CustomAssetFactory = ${function:CustomAssetFactory};
+    DaprReleaseJsonUrl    = "https://gitee.com/dapr-cn/dapr-bin-mirror/raw/main/cli/releases.json";
+    Version            = <Version>
+}
+$script=iwr -useb https://cdn.jsdelivr.net/gh/dapr/cli/installer/install.ps1;
```

Comment:
Replace URL with `https://cdn.jsdelivr.net/gh/dapr/cli/install/install.ps1` as this URL doesn't work

```suggestion
$script=iwr -useb https://cdn.jsdelivr.net/gh/dapr/cli/install/install.ps1;
```